pkg/kubernetes/api/core/v1/podtemplatespec/podtemplatespec.go

```diff
@@ -214,18 +214,6 @@ func (b *Builder) WithNodeSelectorNew(nodeselectors map[string]string) *Builder
 	return b
 }
 
-func (b *Builder) WithNodeSelectorByValue(nodeselectors map[string]string) *Builder {
```

Comment:
Why is his method removed? Instead, can you add comment to this method?

Reply:
`WithNodeSelectorByValue` and `WithNodeSelectorNew` seem to do the same thing, except `WithNodeSelectorNew` has a length check. Also `WithNodeSelectorByValue` is not used anywhere.

@kmova -- What should I add in the comment for this method?

Reply:
We must have migrated from this function to "WithNodeSelectorNew". So makes sense to remove this.